packer/resources/features/README.md

```diff
@@ -0,0 +1,25 @@
+# Features
```

Comment:
@sihil I didn't know that you were writing this README in your branch as well. Whoever gets merged first wins? :)